templates/blocks/hubspot_signups.twig

```diff
@@ -0,0 +1,10 @@
+<script type="text/javascript">
+	window.addEventListener("message", event => {
```

Comment:
We could check whether the page has a `leadin/hubspot-form-block` block and only include this listener if it does.

Reply:
Though it's a very small listener, probably not worth the effort :shrug: